### ISSUE TITLE
Create React App Template Compilation Error

### ISSUE THREAD

**Issue by di2pra** [twilio-labs/paste#2262]
**Description of the bug : Create React App Template Compilation Error**

I'm getting the following error with the create react app template

```
ERROR in ./node_modules/@twilio-paste/color-contrast-utils/dist/index.es.js 1945:10-17

Should not import the named export 'props' (imported as 'S') from default-exporting module (only default export is available soon)


ERROR in ./node_modules/@twilio-paste/color-contrast-utils/dist/index.es.js 1951:10-17

Should not import the named export 'props' (imported as 'S') from default-exporting module (only default export is available soon)


ERROR in ./node_modules/@twilio-paste/color-contrast-utils/dist/index.es.js 1957:10-17

Should not import the named export 'props' (imported as 'S') from default-exporting module (only default export is available soon)


ERROR in ./node_modules/@twilio-paste/style-props/dist/index.es.js 434:22-29

Should not import the named export 'props' (imported as 'm') from default-exporting module (only default export is available soon)
```

**Comment by shleewhite** [comment]
Hi @di2pra! Thank you for letting us know! This is an issue with our esbuild configuration. We are working on a fix, will update the thread when it is merged.

**Comment by shleewhite** [comment]
@di2pra Unfortunately, it looks like this fix may take some time, so until it is ready you can roll `@twilio-paste/core` back to `10.20.0`.

**Comment by SiTaggart** [comment]
Hi @di2pra 

This is now fixed in `@twilio-paste/core@11.1.2`. Let us know if you encounter anymore bugs, thanks for making Paste better!

**Comment by di2pra** [comment]
thank you ! :)
